index.js

```diff
@@ -604,4 +604,14 @@ Notifications.setNotificationCategories = function() {
   return this.callNative('setNotificationCategories', arguments);
 }
 
+// https://developer.android.com/reference/android/app/NotificationManager#IMPORTANCE_DEFAULT
+Notifications.IMPORTANCE = Object.freeze({
```

Comment:
A small change to follow the current case would be to rename `IMPORTANCE` to `Importance` (ucfirst).
For constant (`MIN`, `LOW`, ....) this is OK.

Reply:
👍 Done